app/page.tsx

```diff
@@ -23,35 +19,25 @@ export default async function Index() {
   const isSupabaseConnected = canInitSupabaseClient()
 
   return (
-    <div className="flex-1 w-full flex flex-col gap-20 items-center">
-      <nav className="w-full flex justify-center border-b border-b-foreground/10 h-16">
-        <div className="w-full max-w-4xl flex justify-between items-center p-3 text-sm">
-          <DeployButton />
-          {isSupabaseConnected && <AuthButton />}
-        </div>
+    <div
+      className={[
+        'flex flex-col items-center justify-center flex-1',
+        'w-full',
+      ].join(' ')}
```

Comment:
Is this an array for formatting purposes? Wondering why this can't just be a string

Reply:
Agree. Pretty sure this is overkill. The classNames on line 26 - 28 are static.

Reply:
Yes, this was done for formatting purposes and DX. It's a personal preference when I'm working with Tailwind to improve readability by keeping related Tailwind classes together while making it fast and easy to add new dynamic classes.

I personally find this syntax vastly preferable to long single strings with awkward template literals sprinkled throughout.

It may seem silly on a simple placeholder component like here, but it has real benefits as components become more complex. For example:
![Code_C3K1gkXL0p](https://github.com/LetsGetTechnical/gridiron-survivor/assets/96756923/eb6dc7a0-206b-48b0-b31a-29c0d0f0e473)